<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OngletProblemes</class>
 <widget class="QWidget" name="OngletProblemes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Relations">
         <property name="text">
          <string>MP3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Physiques">
         <property name="text">
          <string>Physiques</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Id Album</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Id_Album"/>
           </item>
           <item>
            <widget class="QPushButton" name="Ok_Album">
             <property name="text">
              <string>Ok</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Id Titre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Id_Titre"/>
           </item>
           <item>
            <widget class="QPushButton" name="Ok_Titre">
             <property name="text">
              <string>Ok</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="Resultat">
           <property name="text">
            <string>RÃ©sultat</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="TabProbl">
     <property name="columnCount">
      <number>12</number>
     </property>
     <column>
      <property name="text">
       <string>Titre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Album</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Artiste</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Chemin MP3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Num Piste</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Annee</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Support Mp3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Support Phys</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Commentaires</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Id_Titre</string>
      </property>
     </column>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
